src/models/transfer.js

```diff
@@ -116,7 +116,7 @@ PersistentModelMixin(Transfer, sequelize, {
   ledger: Sequelize.STRING(1024),
   debits: JsonField(sequelize, 'Transfer', 'debits'),
   credits: JsonField(sequelize, 'Transfer', 'credits'),
-  part_of_payment: Sequelize.STRING(1024),
+  additional_info: JsonField(sequelize, 'Transfer', 'additional_info'),
```

Comment:
In what sense is this `additional_info` part of the data? You could argue that file metadata is part of the file's data because they are stored together on the disk, but of course that's not the desired interpretation.